gen3/bin/sqs.sh

```diff
@@ -62,6 +62,7 @@ gen3_sqs_create_queue() {
     gen3 cd 1>&2
     cat << EOF > config.tfvars
 sqs_name="$sqsName"
+slack_webhook="$g3k_slack_webhook"
```

Comment:
how to fix queues that have already been created?

Reply:
So I think I would need to rerun the modules manually, but Im not sure if there is a better way to do it.

Reply:
🤔 i mean if we remember to do it in all the environments that's fine - but i think `kube-setup-fence` has probably been run everywhere by now so that's a lot of environments

Reply:
Yea, I have to go through and run some other things in every environment anyway 😔.

Reply:
😨 okay then!